packages/mint-components/src/components/tax-and-cash/sqm-document-type-form/sqm-document-type-form.tsx

```diff
@@ -0,0 +1,162 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, Prop, h } from "@stencil/core";
+import { getProps } from "../../../utils/utils";
+import {
+  DocumentTypeFormView,
+  DocumentTypeFormViewProps,
+} from "./sqm-document-type-form-view";
+import {
+  UseDocumentTypeFormResult,
+  useDocumentTypeForm,
+} from "./useDocumentTypeForm";
+import deepmerge from "deepmerge";
+import { DemoData } from "../../../global/demo";
+import { useParent } from "../../../utils/useParentState";
+import { TAX_CONTEXT_NAMESPACE } from "../sqm-tax-and-cash/data";
+import { isDemo } from "@saasquatch/component-boilerplate";
+
+/**
+ * @uiName Tax Document Type Selection Form
+ */
+@Component({
+  tag: "sqm-document-type-form",
+  shadow: true,
+})
+export class DocumentTypeForm {
+  /**
+   * Sub text shown at the top of the page, used to show the current step of the tax form.
+   * @uiName Tax form step text
+   */
+  @Prop() formStep: string = "Step 3 of 4";
+  /**
+   * Sub text shown above form selection radio buttons
+   * @uiName Tax form selection heading text
+   */
+  @Prop() formLabel: string = "Select a tax form";
+  /**
+   * Heading text shown at the top of the top of page
+   * @uiName Tax form heading text
+   */
+  @Prop() taxForm: string = "Tax form";
+  /**
+   * Label text for the W9 radio button
+   * @uiName W9 radio button label
+   */
+  @Prop() w9Label: string = "W9";
+  /**
+   * Subtext for the W9 radio button
+   * @uiName W9 radio button subtext
+   */
+  @Prop() w9Description: string =
+    "W9 For participants based in the US, joining the referral program of a US-based company.";
+  /**
+   * Label text for the W8 radio button
+   * @uiName W8 radio button label
+   */
+  @Prop() w8Label: string = "W8-Ben";
+  /**
+   * Subtext for the W8 radio button
+   * @uiName W8 radio button subtext
+   */
+  @Prop() w8Description: string =
+    "W8-BEN For individuals residing outside of the US, joining the referral program of a US-based company.";
+  /**
+   * Label text for the W8E radio button
+   * @uiName W8E radio button label
+   */
+  @Prop() w8ELabel: string = "W8-BEN-E";
+  /**
+   * Subtext for the W8E radio button
+   * @uiName W8E radio button subtext
```

Comment:
We should use the full name "W-8 BEN" / "W-8 BEN-E" here (any anywhere else we have user-facing labels/descriptions